articles/how-to-guides/create-a-designer-extension/create-a-designer-extension.md

```diff
@@ -0,0 +1,71 @@
+---
+uid: how-to-guides.create-a-designer-extension
+note: This one will only cover the new element creation. We will need to write one that will form part of 'creating a new designer' that will allow you to define your visuals too.
+---
+# Create a Designer Extension
+
+If you have worked in Intent Architect's available set of designers such as the Services or the Domain designer, chances are you might have asked yourself the question whether you could change them to some extent.
+In this how-to you will learn how to add your own Element to a Domain designer (or any other designer) and be able to reference it in the designer as if it is an actual type and be able to generate content from that new element.
+
+Create a new Module Builder application and give it the name `DomainDesignerExtension`.
+
+![Domain Design Extension Creation](images/design-extension-creation.png)
+
+And select the `Module Builder - C#` option and click on `Create`.
+
+![Domain Design Extension Customize](images/design-extension-customize.png)
+
+Once the installation dialog is complete you can click on `Close`.
+
+![Installation Dialog](images/installation-dialog.png)
+
+Click on the `Module Builder` designer and create a package with the name `MyDomainExtension`.
+
+After that, go ahead and install the metadata for the Domain designer.
+
+<p><video style="max-width: 100%" muted="true" loop="true" autoplay="true" src="videos/install-domain-module-metadata.mp4"></video></p>
+
+Return to the `Module Builder` designer and setup the following:
+
+Reference the Domain designer in the `MyDomainExtension` package.
+This will make our elements bundled with the `MyDomainExtension` package and targeting the `Domain` designer.
```

Comment:
Either add a line above this one for it to be a paragraph break or alternatively remove the line break.

This sentence is broken, it needs words added/removed/changed.